io/plugins/eu.esdihumboldt.hale.io.topojson.test/src/eu/esdihumboldt/hale/io/topojson/test/TopoJsonInstanceWriterTest.groovy

```diff
@@ -138,6 +134,14 @@ class TopoJsonInstanceWriterTest {
 				date null
 				geometry(createGeometry('POLYGON ((20 10, 40 10, 40 20, 20 20, 20 10))', 4326))
 			}
+
+			SimpleType {
+				id '4'
+				name 'Area 4\u0000'
+				label 'null'
+				date badDate
```

Comment:
Why did you remove the test with the valid date? I think it would be still good to test a valid date as well. The value is already null on the other instances, so how the null date is handled can be tested on them.